language/bert/quantization/calibrate.py

```diff
@@ -0,0 +1,182 @@
+import argparse
+import json
+import os
+import pickle
+import sys
+
+sys.path.insert(0, os.getcwd())
+
+import torch
+import yaml
+from torch.utils.data import DataLoader
+from transformers import BertConfig, BertForQuestionAnswering
+
+import model_compressor  # isort:skip
+
+from quantization.custom_symbolic_trace import custom_symbolic_trace  # isort:skip
+from quantization.utils import random_seed, set_optimization  # isort:skip
+
+
+def load_pytorch_model(model_path, model_config_path, use_gpu):
+    with open(model_config_path) as f:
+        config_json = json.load(f)
+
+    config = BertConfig(
+        attention_probs_dropout_prob=config_json["attention_probs_dropout_prob"],
+        hidden_act=config_json["hidden_act"],
+        hidden_dropout_prob=config_json["hidden_dropout_prob"],
+        hidden_size=config_json["hidden_size"],
+        initializer_range=config_json["initializer_range"],
+        intermediate_size=config_json["intermediate_size"],
+        max_position_embeddings=config_json["max_position_embeddings"],
+        num_attention_heads=config_json["num_attention_heads"],
+        num_hidden_layers=config_json["num_hidden_layers"],
+        type_vocab_size=config_json["type_vocab_size"],
+        vocab_size=config_json["vocab_size"],
+    )
```

Comment:
`config_json`이 사전이라면, 다음이 동작하는지 확인해서 적용 부탁드립니다.

```suggestion
    config = BertConfig(**config_json)
```

여기 뿐만 아니라 설정을 읽어들여서 설정을 만드는 다른 부분들도 비슷한 방식으로 지금보다 간명하게 기술할 수 있을지 검토 바랍니다.

Reply:
https://github.com/furiosa-ai/inference/pull/12/commits/a9b6128656620f22fec2e635960b67191f119bf6 반영했습니다. 
https://github.com/furiosa-ai/inference/pull/12/commits/6ff06fb23e12eae75ffacb1b099dd439e33d0b73 반영했습니다.

Reply:
다음은 추측에 기반해서 작성하였습니다. 정래 님이 생각하신 `get_kwargs` 역할에 맞는지 확인하고, 맞으면 적용 부탁드립니다:

```python
import inspect
from typing import Any, Mapping


def get_kwargs(fn, config_dict: Mapping[str, Any]):
    params = inspect.signature(fn).parameters
    return {
        k: v
        for k, v in conifg_dict
        if k in params and params[k].annotation in (inspect.Parameter.emtpy, type(v))
    }
```

Reply:
[review: simpler is better](https://github.com/furiosa-ai/inference/pull/12/commits/a0e352be5952ff6f2e7711768be88a91bc3601db) 반영했습니다. 훨씬 깔끔해졌네요.